frontend/src/views/ManagementPage/index.vue

```diff
@@ -38,32 +39,52 @@ import UserTools from "./UserTools"
 import TeledeclarationBanner from "./TeledeclarationBanner"
 import CanteenCreationDialog from "./CanteenCreationDialog"
 import ActionsBanner from "./ActionsBanner"
+import SuccessBanner from "./SuccessBanner"
 import validators from "@/validators"
+import { lastYear } from "@/utils"
 
 export default {
+  name: "ManagementPage",
   components: {
     CanteensPagination,
     UserTools,
     PageSatisfaction,
     TeledeclarationBanner,
     ActionsBanner,
+    SuccessBanner,
     CanteenCreationDialog,
   },
   data() {
     return {
       validators,
       canteenCount: undefined,
-      showTeledeclarationBanner: true,
+      hasActions: false,
       showCanteenCreationPrompt: null,
+      showTeledeclarationBanner: true,
```

Comment:
```suggestion
      teledeclarationCampaignActive: true,
```
Peut-être avec bcp de `showX` c'est plus lisible de rendre les variables plus descriptifs ?

Reply:
Fix ici : https://github.com/betagouv/ma-cantine/pull/2701/commits/d27b82527a2535222944c16525c4b84f0ff63bd2